frontend/src/components/containers/ContainersDetailContent.tsx

```diff
@@ -74,6 +70,9 @@ const ContainersDetailContent = () => {
         extra={
           !isLoaded ? null :
             <Space>
+              <Button onClick={() => navigate(
```

Comment:
There goes another comment ... I meant the containers that are unlikely to be useful for mass queueing. A rack or a box are not sample holding but they can be used to queue samples in tubes. Queueing all sample in a freezer is unlikely to be a good option. Buut let this go i guess the empty list should be enough to pass the message ;)